### ISSUE TITLE
[Bug] NavigationPages as TabbedPage Children

### ISSUE THREAD

**Issue by Phenek** [xabre/xamarin-forms-tab-badge#58]
Hello,

Setting Badges on Tabbed NavigationPages is showing the logo and text but no badges:
```
            var page = new SearchPage();
            var navigation = new NavigationPage(page)
            {
                Title = "Search",
                Icon = "Search",
            };

            navigation.SetBinding(TabBadge.BadgeTextProperty, nameof(BottomMenuViewModel.Count));
            navigation.SetBinding(TabBadge.BadgeColorProperty, nameof(BottomMenuViewModel.Color));
            navigation.SetBinding(TabBadge.BadgeTextColorProperty, nameof(BottomMenuViewModel.TextColor));
            navigation.BindingContext = viewModel;
            Children.Add(navigation);
```

Setting Badges, Icon & text on first stacked ContentPage is showing badge but no text & no Icon.
```
            var tab = new SearchPage()
            {
                Title = "Search",
                Icon = "Search",
            };
            var navigation = new NavigationPage(tab);

            tab.SetBinding(TabBadge.BadgeTextProperty, nameof(BottomMenuViewModel.Count));
            tab.SetBinding(TabBadge.BadgeColorProperty, nameof(BottomMenuViewModel.Color));
            tab.SetBinding(TabBadge.BadgeTextColorProperty, nameof(BottomMenuViewModel.TextColor));
            tab.BindingContext = viewModel;
            Children.Add(navigation);
```
Mixing Both works for iOS but on Android (Bottom position) it isn't showing the badge
```
            var tab = new AccountPage();
            var navigation = new NavigationPage(tab)
            {
                Title = "Profil",
                Icon = "Profile",
            };

            tab.SetBinding(TabBadge.BadgeTextProperty, nameof(BottomMenuViewModel.Count));
            tab.SetBinding(TabBadge.BadgeColorProperty, nameof(BottomMenuViewModel.Color));
            tab.SetBinding(TabBadge.BadgeTextColorProperty, nameof(BottomMenuViewModel.TextColor));
            tab.BindingContext = viewModel;
            Children.Add(navigation);

```
The NavigationPage should managed the badges not its first stacked ContentPage, don't you think?
Regards,

**Comment by xabre** [comment]
v2.1.1 realeased with fix for #63 and #58. Give it a try and let me know.

**Comment by Phenek** [comment]
👍  Nice Thanks!
